packages/web-component/src/lib/descope-wc/BaseDescopeWc.ts

```diff
@@ -182,6 +183,16 @@ class BaseDescopeWc extends HTMLElement {
     this.#flowState.update({ stepId, executionId });
   }
 
+  #syncStateWithVisibility() {
+    if (!document.hidden) {
+      // Defer the update a bit, it won't work otherwise
+      setTimeout(() => {
+        // Trigger state update that will redirect and pending deferred redirection
+        this.#flowState.update({ deferredRedirect: false });
+      }, 300);
```

Comment:
I know we talked about it, and I don't have a better idea how we can solve it, 
but putting here an arbitrary delay is risky

Reply:
It's the best we have at the moment. I'm not aware of a different event we can listen too, I think it's Android OS related. In any case it's only for Android flows.